### ISSUE TITLE
Error uploading or enumerating designs - Error 500 Internal Server Error

### ISSUE THREAD

**Issue by geoffbaldwin** [SynBioDex/SBOLCanvas#349]
When uploading or enumerating an error 500 internal server error is reported. This can be resolved by running SBOL Canvas from an incognito browser, if that fails then reloading SBOL canvas in a new incognito window seems to again resolve the issue.
This became apparent in a class when everyone had issues with upload and enumeration.

![image](https://github.com/user-attachments/assets/3b0bc2d4-f08a-46df-bfa4-41800ea110db)

**Comment by Drock54651** [comment]
If you were to enumerate or upload the same design again in a normal browser, would it still result in an error? I'm having trouble reproducing this issue.

**Comment by geoffbaldwin** [comment]
Yes, at the time of having problems the normal browser continually fails. This error seems to emerge on repeated usage. When running a class, by the end of the class everyone was having this problem. I've just come in this morning and my old browser page was still open, it now seems to be uploading fine. Since this is a server error could this be something to do with overload at the server end.

**Comment by geoffbaldwin** [comment]
It might be worth checking your server logs - incidences were occuring on 11th Nov at 11-12 am UK time.

**Comment by cjmyers** [comment]
It sounds like maybe SBOLCanvas has an issue communicating with the backend.  We recently split the frontend and backend deployments, so I'm wondering if this is related somehow.

**Comment by cjmyers** [comment]
We cannot reproduce this one.  Is it still happening?

**Comment by geoffbaldwin** [comment]
I've not been using it intensively enough to find out. Students should be using this for their coursework, so I will encourage them to comment here if they experience the issues.
